addons/hr_homeworking/views/hr_employee_views.xml

```diff
@@ -21,7 +21,7 @@
             </xpath>
             <xpath expr="//group[@name='departure']" position="after">
                 <group string="Homeworking">
-                    <span class="text-muted fst-italic oe_inline" colspan="2">Specify your default work location for each day of the week. This schedules will repeat itself each week.</span>
+                    <span class="text-muted fst-italic oe_inline" colspan="2">Specify your default work location for each day of the week. This schedule will repeat itself each week.</span>
```

Comment:
seems like there is no pot file for hr_homeworking yet.. otherwise we would have needed to change it too